ci-operator/config/openshift/build-test-images/openshift-build-test-images-master.yaml

```diff
@@ -0,0 +1,35 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-openshift-4.6
+images:
+- context_dir: roots2i
+  dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    src:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/build-test-images/roots2i/.
```

Comment:
i don't think you need this "inputs" section

Reply:
i'm confused ... how else does it know to clone from `github.com/openshift/build-test-images` ?

Reply:
it knows it by the dir structure.  See: 
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/sippy/openshift-sippy-master.yaml

Reply:
interesting ... I'll give it a go then